src/modules/appointments/services/appointment.service.ts

```diff
@@ -17,16 +18,87 @@ export const createAppointment = async (
 };
 
 export const getUserCreatedAppointments = async (payload: any) => {
```

Comment:
Let's create a type for the payload request to remove `any` type.

Reply:
fixed!